### PR TITLE
Make Author, Role & Topic case sensitive

### PR DESCRIPTION
**Pull request by stephanheilner**

### CODE REVIEW

File: LDSContent/ItemPackage.swift [hunk 1]
```diff
@@ -566,7 +566,7 @@ extension ItemPackage {
     }
     
     public func authorWithGivenName(givenName: String, familyName: String) -> Author? {
-        return db.pluck(AuthorTable.table.filter(AuthorTable.givenName.lowercaseString == givenName.lowercaseString && AuthorTable.familyName.lowercaseString == familyName.lowercaseString)).map { AuthorTable.fromRow($0) }
```
Comment (hiltonc) [inline LDSContent/ItemPackage.swift]: For authors, roles, and topics, I think we'd be better off if we didn't even do `lowercaseString`, but instead let everything through. If they end up getting duplicate topics, well, they should normalize their content. We can provide warnings that they're getting through, but we should encourage them to have the purest content possible.

**Comment by nickmshelley** [comment]
👍 

Not sure if I should merge or not so I'm just going to 👍  and let someone else do it.

**Comment by hiltonc** [comment]
w00t!

**Comment by hiltonc** [comment]
👍

**Comment by stephanheilner** [comment]
I need to fix the tests, and probably rename this PR.

**Pull request merged**